app/models/mailing_list/wizard.rb

```diff
@@ -17,9 +17,9 @@ class Wizard < ::GITWizard::Base
       Steps::AlreadySubscribed,
       Steps::ReturningTeacher,
       Steps::AlreadyQualified,
+      Steps::Subject,
```

Comment:
Is this one just changing the mailing list @emmaachesongray ?

We do want to do this but not yet - we need to make other changes in the mailing list first :-) 

Can you remove this change?